docs/run_test/run.md

```diff
@@ -0,0 +1,23 @@
+# Run LISA
+
+- ## [Quick start](quick_run.md)
+  - ### [Getting started with Azure](quick_run.md)
+- ## [Learn Microsoft tests](microsoft_tests.md)
+  - ### [How to run](microsoft_tests.md#how-to)
+  - ### [Test priority](microsoft_tests.md#test-priority)
+  - ### [Test tier](microsoft_tests.md#test-tier)
+  - ### [Test cases specification](microsoft_tests.md#test-cases-specification)
+  - ### [LISAv2 legacy tests](run_legacy.md)
+    - [Preparation](run_legacy.md#preparation)
+    - [Limitations](run_legacy.md#limitation)
+    - [Roadmap](run_legacy.md#roadmap)
+  - ### [Table of test cases](test_table.md)
+- ## [Run tests on different platforms](platform.md)
+  - ### [Azure](platform.md#run-on-azure)
+  - ### [Ready platform](platform.md#run-on-ready-computers)
+- ## [Use runbook](runbook.md)
+  - ### [How to configure](runbook.md#how-to)
+  - ### [Customization](runbook.md#reference)
+- ## [Analyze test results](understand_results.md)
+  - ### [Understand log messages](understand_results.md#understand-log-messages)
```

Comment:
It needs to introduce test results (PASSED, FAILED, ...). The "understand log messages" shouldn't be part of analyze test results.